src/navigation/MainScreen.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {createStackNavigator} from '@react-navigation/stack';
+import {NavigationContainer} from '@react-navigation/native';
+import {Login} from '../screens/login';
+import {FlatListToDoList} from '../screens/toDoList';
+import {NavigationParam} from '.';
```

Comment:
```suggestion
import {NavigationParam} from '.types';
```
This can cause circular dependency.